lib/fluent/plugin_helper/server.rb

```diff
@@ -251,6 +251,7 @@ def server_create_for_tls_connection(shared, bind, port, conf, backlog, socket_o
         :generate_cert_country, :generate_cert_state, :generate_cert_state,
         :generate_cert_locality, :generate_cert_common_name,
         :generate_cert_expiration, :generate_cert_digest,
+        :ensure_fips
```

Comment:
[nits] If you add `,` at last,  we can easily add items next time.

```suggestion
        :ensure_fips,
```

Reply:
Thanks!